### PR TITLE
core models and its tests added

### PR DESCRIPTION
**Pull request by anarbyrm**

### CODE REVIEW

File: cargo_backend/core/models.py [hunk 1]
```diff
@@ -1,9 +1,76 @@
...
+    def save(self, *args, **kwargs):
+        self.full_clean()
+        super().save(*args, **kwargs)
+
+
+class News(models.Model):
+    title = models.CharField(max_length=255)
+    content = models.TextField()
+    image = models.ImageField(upload_to="news/images")
+    date_published = models.DateTimeField(auto_now_add=True)
```
Comment (mmzeynalli) [inline cargo_backend/core/models.py:56]: Try to stick created_at, updated_at fields. No need to modify them to different names

File: requirements.txt [hunk 2]
```diff
@@ -2,3 +2,5 @@ django>=4.1.0, <4.2.0
 djangorestframework>=3.14.0, <3.15.0
 psycopg2-binary==2.9.5
 factory-boy==3.2.1
+pillow==9.4.0
+
```
Comment (mmzeynalli) [inline requirements.txt:6]: pillow?

**Review by mmzeynalli — CHANGES_REQUESTED** [review]

**Comment by mmzeynalli** [comment]
It would also be better to have file structure list this

tests/
  - factories/
     - core
     - account
   tests.py

File: requirements.txt (hunk 2 above)
Comment (anarbyrm) [reply requirements.txt:6]: for image field in news model

File: cargo_backend/core/models.py (hunk 1 above)
Comment (anarbyrm) [reply cargo_backend/core/models.py:56]: do you want to say that only created_at field is enough?

Comment (mmzeynalli) [reply cargo_backend/core/models.py:56]: Instead of date_pblished -> created_at, date_edited -> updated_at

Comment (anarbyrm) [reply cargo_backend/core/models.py:56]: roger

**Review by mmzeynalli — APPROVED** [review]

**Pull request merged**